### ISSUE TITLE
Use the metrics of extra charting in live analysis

### ISSUE THREAD

**Issue by Siderite** [Siderite/lichessTools#165]
More information during live analysis might be worth it. The Principled metric could give some insights to the player during analysis or explorer practice. It could maybe be coupled with the computer eval or have its own toggling mechanism.

Other possibilities open for live analysis that don't have place in the game analysis chart:
- worst/best piece
- square control
- tension of current position
- some heat mapping of the board squares
- generate smileys in real time perhaps

**Comment by Siderite** [comment]
add a 
```
<tick style="
  height: 10px;
  background: green;
  top: 263px;"></tick>

```
to div.eval-gauge

**Comment by Siderite** [comment]
Material and Principled metrics implemented in v2.1.5
